src/quo/components/profile/collectible_list_item/view.cljs

```diff
@@ -67,108 +60,129 @@
            {:size   :paragraph-1
             :weight :semi-bold
             :style  style/card-detail-text}
-           collectible-name]]
+           collectible-name]])
 
-         :else
-         [:<>
-          [collection-avatar/view
-           {:size  :size-20
-            :image avatar-image-src}]
-          [rn/view {:style {:width 8}}]
-          [text/text
-           {:size            :paragraph-1
-            :weight          :semi-bold
-            :ellipsize-mode  :tail
-            :number-of-lines 1
-            :style           style/card-detail-text}
-           collectible-name]])])
+   [rn/view
+    {:style (style/avatar-container (:avatar-loaded? state))}
+    (when-not (:avatar-error? state)
+      [:<>
+       [collection-avatar/view
+        {:size        :size-20
+         :on-load-end #(set-state (assoc state :avatar-loaded? true))
+         :on-error    #(set-state (assoc state :error? true))
+         :image       avatar-image-src}]
+       [rn/view {:style {:width 8}}]])
+    [text/text
+     {:size            :paragraph-1
+      :weight          :semi-bold
+      :ellipsize-mode  :tail
+      :number-of-lines 1
+      :style           style/card-detail-text}
+     collectible-name]]])
 
 (defn- card-view
-  [{:keys [avatar-image-src collectible-name community? counter
-           gradient-color-index image-src status]}]
+  [{:keys [avatar-image-src collectible-name community? counter state set-state
+           gradient-color-index image-src collectible-supported?]}]
```

Comment:
what is collectible-supported? is that the file type? perhaps file-type-supported? is better explained? 
no strong opinions on this though

Reply:
perhaps using `unsupported-file?` is clearer than those?

Reply:
I renamed it to `supported-file?`